### ISSUE TITLE
Light color theme (new website) not working? 

### ISSUE THREAD

**Issue by kexul** [michellab/Sire#423]
Dear Sire developers: 
The new website has three color themes, but the light color theme does not seem to work. 
![website](https://user-images.githubusercontent.com/5920217/211787886-4712eaa6-c811-470b-ac95-5b7b2fb8d038.gif)

**Comment by kexul** [comment]
I thought these icons means `light`, `dawn`, and `dark`. The html code says it's `light`, `auto`, and `dark`, so it's an intended behavior?

**Comment by chryswoods** [comment]
Thanks - Yes - this is intended behaviour. There are two themes; 'light' (the proper OpenBioSim theme) and 'dark' (the furo dark theme, in case you don't like purple). The 'auto' theme just sets the theme based on your browser (whether you have system-wide dark mode enabled or not).

**Comment by kexul** [comment]
Hi @chryswoods , would you mind adding a theme with white background and black text? 🙏 🙏 🙏

**Comment by chryswoods** [comment]
Yes - I think I should be able to do this, but probably not before the 2023.1.0 release this week. It will involve either adding a third theme to furo, or me taking the furo dark theme and modifying that to create a lighter version of the openbiosim theme. The openbiosim theme is the one that you will soon see used on openbiosim.org.

**Comment by kexul** [comment]
Many thanks for your work! Looking forward to seeing that! 😃
